TeamCharles/workforce-management
# 46-shared-layout :: added layout template and additional required files for it to render

## Status
**READY**

## Documentation

[ ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[X] This PR does **NOT** require documentation.

## Description
Added layout view and fixed bower components location.

## Resolves Issue Number
#46 

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

```sh
git pull --prune
git checkout 46-shared-layout
dotnet run
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1.  DELETE `bower_components`
2.  Run `bower install`
3.  Run `dotnet run`
4.  Navigate to localhost in browser (Should see empty layout view)

## Impacted Files in Application
List general components of the application that this PR will affect:

* /Views/_ViewStart.cshtml
* /Views/_ViewImports.cshtml
* /Views/Shared/_NavbarView.cshtml
* /Views/Shared/_Layout.cshtml
* /.bowerrc
